docs/source/troubleshooting/index.rst

```diff
@@ -0,0 +1,79 @@
+.. _troubleshooting:
+
+===============
+Troubleshooting
+===============
+
+.. contents::
+   :local:
+
+.. _clearing-schema-mutation-lock:
+
+Clearing the schema mutation lock
+=================================
+
+While performing schema mutations (e.g. creating or dropping tables) in Cassandra, we hold a :ref:`schema mutation lock <schema-mutation-lock>`.
+If an AtlasDB client dies while holding the lock, the lock must be manually cleared or clients will not be able to perform schema mutations.
+Prior to AtlasDB 0.19, we would always grab the schema mutation lock on startup, and thus would fail to start until the lock had been cleared.
+
+The error message you see when hitting this issue is the following.
+
+.. code-block:: none
+
+   We have timed out waiting on the current schema mutation lock holder. We have
```

Comment:
I actually think both of them will show up now. Just added the heartbeat exception message as well so they're both on the docs (some people will have see the message pre-heartbeat only if they're on older versions).